src/components/Workflow/index.jsx

```diff
@@ -2,13 +2,14 @@ import React from 'react';
 import ThemedImage from '@theme/ThemedImage';
 import useBaseUrl from '@docusaurus/useBaseUrl';
 import styles from './styles.module.css';
+import Link from '@docusaurus/Link';
 
 export default function Workflow() {
     return (
         <div className={[styles.workflow, styles.section_padding].join(' ')}>
             <div className={styles.how_oras_works}>
                 <h1>How ORAS Works?</h1>
-                <p>Able to distribute any artifacts across OCI-compliant registries or OCI Image Layout. See all <a href="/docs/implementors/#registries-supporting-oci-artifacts">compatible registries</a>.</p>
+                <p>Able to distribute any artifacts across OCI-compliant registries or OCI Image Layout. See all <Link to="/docs/implementors/#registries-supporting-oci-artifacts">compatible registries</Link>.</p>

```

Comment:
```suggestion
                <p>Able to distribute any artifacts across OCI-compliant registries or OCI Image Layout. See all <Link to="/adopters">compatible registries</Link>.</p>

```

I would change this link to adopters page as #157 introduces a page to list all the registries. Hold this PR, until that gets merged.

Reply:
Yeah, sure